packages/sui-bundler/webpack.config.client.dev.js

```diff
@@ -83,7 +85,10 @@ const webpackConfig = {
     }),
     new WebpackManifestPlugin({fileName: 'asset-manifest.json'}),
     new webpack.HotModuleReplacementPlugin(),
-    new ReactRefreshWebpackPlugin({overlay: false})
+    new ReactRefreshWebpackPlugin({overlay: false}),
+    new HtmlWebpackInjectAttributesPlugin({
+      crossorigin: 'anonymous'
```

Comment:
This is necessary to make it work. The hydration mismatch overlay only shows up when it detects errors containing the words hydration or hydrating (see here https://github.com/BuilderIO/hydration-overlay/blob/661aa0e17a50777fe9d7222fe0759d84798b1c3e/packages/lib/src/hydration-overlay-initializer.js#L5 ). Since the app.js script is being served from localhost:8080 whereas the dev server is running on localhost:3000, an error from app.js would be coming from a different origin and pop up as generic Script error, which would not be captured by hydration-overlay. Hence the requirement for a new dependency html-webpack-inject-attributes-plugin.